plugins/haconfig/keepalived/plugin.go

```diff
@@ -0,0 +1,147 @@
+package keepalived
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	"github.com/openshift/origin/pkg/generate/app"
+	"github.com/openshift/origin/pkg/haconfig"
+)
+
+//  HA configurator plugin for keepalived sidecar.
+type KeepAlivedPlugin struct {
```

Comment:
It's ugly both ways but this way you don't have the A poking you in the eye.

----- Original Message -----
> > +	"os"
> > +	"strings"
> > +
> > +	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
> > +	"github.com/golang/glog"
> > +
> > +	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
> > +	configcmd "github.com/openshift/origin/pkg/config/cmd"
> > +	"github.com/openshift/origin/pkg/generate/app"
> > +	"github.com/openshift/origin/pkg/haconfig"
> > +)
> > +
> > +//  HA configurator plugin for keepalived sidecar.
> > +type KeepAlivedPlugin struct {
> 
> Will change that - yeah, was debating over that spelling as well - changed it
> back and forth.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1807/files#r28802654